### DISCUSSION TITLE
AllTubeDownload doesn’t work

### DISCUSSION THREAD

**Discussion by philact3** [IceWhaleTech/CasaOS#324] [Q&A]
Hi folks!
Installed AlltubeDownloads, but it doesn't work.
The icon has a red point.

How i can resolve?

Tia everybody

**Comment by philact3** [comment]
Anybody?

**Reply by Lauren-ED209** [reply]
the red poiint means the docker is stop running. so try restart the service. 
feel like it's an issue. not a discussion. create an issue if it's necessary.

**Reply by jerrykuku** [reply]
Alltubedownload only works on x86（amd64）device, if you are using arm device. It will not work.This is an issue.
